src/Resources/config/admin.php

```diff
@@ -126,14 +126,22 @@
             ])
             ->call('setTranslationDomain', ['SonataPageBundle'])
 
-        ->set('sonata.page.admin.extension.snapshot', CreateSnapshotAdminExtension::class)
+        ->set(CreateSnapshotAdminExtension::class)
             ->tag('sonata.admin.extension', ['target' => 'sonata.page.admin.page'])
             ->tag('sonata.admin.extension', ['target' => 'sonata.page.admin.block'])
             ->args([
                 new ReferenceConfigurator('sonata.page.service.create_snapshot'),
             ])
 
+        ->alias(PageAdmin::class, 'sonata.page.admin.page')
+
         ->alias(BlockAdmin::class, 'sonata.page.admin.block')
```

Comment:
Hey @VincentLanglet 

is it fine we use the `class` as a service and `the string` as an alias like this?

Reply:
This is not done this way for other bundles https://github.com/sonata-project/SonataMediaBundle/blob/4.x/src/Resources/config/doctrine_orm_admin.php

And as far I remember there is a reason. I think this is not recommended for libraries.
So I would let this like it was before.
Maybe @jordisala1991 knows more about it.